public/app/features/alerting/unified/components/contact-points/ContactPoints.v2.tsx

```diff
@@ -22,47 +22,66 @@ import {
   TabContent,
   Tab,
   Pagination,
+  Button,
 } from '@grafana/ui';
-import { contextSrv } from 'app/core/core';
 import ConditionalWrap from 'app/features/alerting/components/ConditionalWrap';
-import { isOrgAdmin } from 'app/features/plugins/admin/permissions';
 import { receiverTypeNames } from 'app/plugins/datasource/alertmanager/consts';
 import { GrafanaManagedReceiverConfig } from 'app/plugins/datasource/alertmanager/types';
 import { GrafanaNotifierType, NotifierStatus } from 'app/types/alerting';
 
+import { AlertmanagerAction, useAlertmanagerAbility } from '../../hooks/useAbilities';
 import { usePagination } from '../../hooks/usePagination';
 import { useAlertmanager } from '../../state/AlertmanagerContext';
 import { INTEGRATION_ICONS } from '../../types/contact-points';
-import { getNotificationsPermissions } from '../../utils/access-control';
-import { GRAFANA_RULES_SOURCE_NAME, isVanillaPrometheusAlertManagerDataSource } from '../../utils/datasource';
+import { GRAFANA_RULES_SOURCE_NAME } from '../../utils/datasource';
 import { createUrl } from '../../utils/url';
+import { Authorize } from '../Authorize';
+import { GrafanaAlertmanagerDeliveryWarning } from '../GrafanaAlertmanagerDeliveryWarning';
 import { MetaText } from '../MetaText';
+import MoreButton from '../MoreButton';
 import { ProvisioningBadge } from '../Provisioning';
 import { Spacer } from '../Spacer';
 import { Strong } from '../Strong';
+import { GrafanaReceiverExporter } from '../export/GrafanaReceiverExporter';
+import { GrafanaReceiversExporter } from '../export/GrafanaReceiversExporter';
 import { GlobalConfigAlert } from '../receivers/ReceiversAndTemplatesView';
 import { UnusedContactPointBadge } from '../receivers/ReceiversTable';
+import { ReceiverMetadataBadge } from '../receivers/grafanaAppReceivers/ReceiverMetadataBadge';
+import { ReceiverPluginMetadata } from '../receivers/grafanaAppReceivers/useReceiversMetadata';
 
 import { MessageTemplates } from './MessageTemplates';
 import { useDeleteContactPointModal } from './Modals';
-import { RECEIVER_STATUS_KEY, useContactPointsWithStatus, useDeleteContactPoint } from './useContactPoints';
-import { ContactPointWithStatus, getReceiverDescription, isProvisioned, ReceiverConfigWithStatus } from './utils';
+import {
+  RECEIVER_META_KEY,
+  RECEIVER_PLUGIN_META_KEY,
+  RECEIVER_STATUS_KEY,
+  useContactPointsWithStatus,
+  useDeleteContactPoint,
+} from './useContactPoints';
+import { ContactPointWithMetadata, getReceiverDescription, isProvisioned, ReceiverConfigWithMetadata } from './utils';
 
 enum ActiveTab {
   ContactPoints,
   MessageTemplates,
 }
 
-const DEFAULT_PAGE_SIZE = 25;
+const DEFAULT_PAGE_SIZE = 10;
```

Comment:
Contact points take up more vertical height, so this seemed like a more appropriate page size.

Reply:
Can we add your comment to the code so we remember why the number is that low?